aea/configurations/constants.py

```diff
@@ -28,11 +28,11 @@
 from aea.crypto.helpers import PRIVATE_KEY_PATH_SCHEMA
 
 
-DEFAULT_CONNECTION = PublicId.from_str("fetchai/stub:0.11.0")
-DEFAULT_PROTOCOL = PublicId.from_str("fetchai/default:0.7.0")
-SIGNING_PROTOCOL = PublicId.from_str("fetchai/signing:0.5.0")
-STATE_UPDATE_PROTOCOL = PublicId.from_str("fetchai/state_update:0.5.0")
-DEFAULT_SKILL = PublicId.from_str("fetchai/error:0.7.0")
+DEFAULT_CONNECTION = PublicId.from_str("fetchai/stub:0.12.0")
+DEFAULT_PROTOCOL = PublicId.from_str("fetchai/default:0.8.0")
+SIGNING_PROTOCOL = PublicId.from_str("fetchai/signing:0.6.0")
+STATE_UPDATE_PROTOCOL = PublicId.from_str("fetchai/state_update:0.6.0")
+DEFAULT_SKILL = PublicId.from_str("fetchai/error:0.8.0")
```

Comment:
let's point to latest, rather than specific versions!

Reply:
That might be a good idea, but I'd suggest doing it in a separate PR.